src/library/scala/runtime/RichInt.scala

```diff
@@ -37,10 +37,6 @@ final class RichInt(val self: Int) extends AnyVal with ScalaNumberProxy[Int] wit
   override def min(that: Int): Int = math.min(self, that)
   override def signum: Int         = math.signum(self)
 
-  /** There is no reason to round an `Int`, but this method is provided to avoid accidental loss of precision from a detour through `Float`. */
-  @deprecated("this is an integer type; there is no reason to round it.  Perhaps you meant to call this on a floating-point value?", "2.11.0")
-  def round: Int = self
```

Comment:
The comment here suggests that this is a deprecation meant to stay forever. IIUC, without it the following silently loses precision:
```scala
val x: Int = ???
x.round
```
because it turns `x` into a `Float` to be able to call `RichFloat.round`, or something like that.

We have similar intended deprecated-forever methods in Scala.js: https://github.com/scala-js/scala-js/blob/v0.6.14/library/src/main/scala/scala/scalajs/js/JSNumberOps.scala#L90-L104